integration-cli/docker_cli_run_unix_test.go

```diff
@@ -338,3 +338,18 @@ func (s *DockerSuite) TestStopContainerSignal(c *check.C) {
 		c.Fatalf("Expected `exit trapped` in the log, got %v", out)
 	}
 }
+
+func (s *DockerSuite) TestRunSwapMoreThanMemoryLimit(c *check.C) {
+	testRequires(c, NativeExecDriver)
```

Comment:
There's not need for this, lxc supports this and this tests won't even arrive to the execdriver